### PR TITLE
Add Detokenization policy to the tokenization changes

### PR DESCRIPTION
**Pull request by fermujica**
This pull request adds the detokenization policy documentation to the tokenization policy articles.
This way, after this is approved, I can merge all tokenization/detokenization changes in one merge.

### CODE REVIEW

File: modules/ROOT/pages/policy-mule4-detokenization.adoc [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+!===
+
+|===
+
+[NOTE]
+--
+This policy is designed to work only for Anypoint Runtime Fabric deployments.
+--
+
+== Tokenization
```
Comment (LorenaSoledad) [inline modules/ROOT/pages/policy-mule4-detokenization.adoc]: Detokenization, maybe? I don't know if maintaining the `Tokenization` title is on purpose or not.

File: modules/ROOT/pages/policy-mule4-detokenization.adoc [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+* To create and apply policies you need the *Manage Policies* permission.
+* You need to have a tokenization service up and running.
+** To configure and see the tokenization service, you need the *Manage Tokenization Services* permission. +
+See xref:anypoint-security::tokenization-example.adoc[Configure and Use Tokenization] for more information.
+
+
+== How Does This Policy Work?
+
+The policy reads a token the payload with the tokenized values. To define which pieces of payload will be detokenized, you need to provide Dataweave 2.0 expressions.
```
Comment (LorenaSoledad) [inline modules/ROOT/pages/policy-mule4-detokenization.adoc]: Please replace Dataweave 2.0 expressions with "selector expressions". We made some last minute modifications and we are not using Dataweave anymore.

File: modules/ROOT/pages/policy-mule4-detokenization.adoc [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+[source,JSON,linenums]
+--
+{
+  "name":"Amun Ra",
+  "socialSecurityNumber":"167-47-9268"
+}
+--
+
+If you want to detokenize the social security number, you can use the following Dataweave expression to specify which piece of information the policy should tokenize:
```
Comment (LorenaSoledad) [inline modules/ROOT/pages/policy-mule4-detokenization.adoc]: Same about Dataweave.

File: modules/ROOT/pages/policy-mule4-detokenization.adoc [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+To learn how to tokenize a value refer to the xref:policy-mule4-tokenization.adoc[Tokenization Policy].
+
+=== Detokenizing Requests
+
+image::policy-mule4-tokenization-67b25.png[]
+
+. A user performs a request to the application protected by the policy.
+. The request is first caught by Anypoint Security, and redirected to the application, reaching first the detokenization policy.
+. The detokenization policy is configured to detokenize the request, so it extracts all the tokens from the payload using the DataWeave 2.0 expressions.
```
Comment (LorenaSoledad) [inline modules/ROOT/pages/policy-mule4-detokenization.adoc]: Same about DW.

File: modules/ROOT/pages/policy-mule4-detokenization.adoc [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+=== Detokenizing Responses
+
+image::policy-mule4-tokenization-9347d.png[]
+
+. A user performs a request to the application protected by the policy.
+. The request is first caught by Anypoint Security, and redirected to the application, reaching first the detokenization policy.
+. The detokenization policy is configured to detokenize the Mule application's response, so it redirects the request without extracting anything from the payload.
+. The Mule application processes the request and returns the result.
+. The detokenization policy extracts all the tokens from the payload using the DataWeave 2.0 expressions.
```
Comment (LorenaSoledad) [inline modules/ROOT/pages/policy-mule4-detokenization.adoc]: Same about DW

File: modules/ROOT/pages/policy-mule4-detokenization.adoc [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+| *Fail on unmatched expressions*
+| If selected, requests will fail when the rules cannot match any token.
+| No.
+
+| *Services*
+| The tokenization services available for the current Organization and Environment. Select the service that you need from the list, remembering that the formats available depends on the service that you choose.
+| Yes.
+
+| *Expression*
+| A DataWeave 2.0 selector expression to be used to extract the information that will be detokenized.
```
Comment (LorenaSoledad) [inline modules/ROOT/pages/policy-mule4-detokenization.adoc]: Same.

File: modules/ROOT/pages/policy-mule4-detokenization.adoc [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+For example:
+[source,dataweave,linenums]
+--
+#[payload.people.creditCardNumber]
+--
+
+Only expressions using dots as field separators are allowed. While `#[payload.people.creditCardNumber]` is valid, `#[payload[‘people’]]` is not.
+
+| No. +
+Keep in mind that if you don't specify a DataWeave expression, the policy will be applied, but won't detokenize anything from your payload.
```
Comment (LorenaSoledad) [inline modules/ROOT/pages/policy-mule4-detokenization.adoc]: Same.

File: modules/ROOT/pages/policy-mule4-detokenization.adoc [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+
+| *Services*
+| The tokenization services available for the current Organization and Environment. Select the service that you need from the list, remembering that the formats available depends on the service that you choose.
+| Yes.
+
+| *Expression*
+| A DataWeave 2.0 selector expression to be used to extract the information that will be detokenized.
+
+For example:
+[source,dataweave,linenums]
```
Comment (LorenaSoledad) [inline modules/ROOT/pages/policy-mule4-detokenization.adoc]: Same

**Review by LorenaSoledad — CHANGES_REQUESTED** [review]

**Review by Cristian-Venticinque — APPROVED** [review]

File: modules/ROOT/pages/policy-mule4-detokenization.adoc (hunk 2 above)
Comment (fermujica) [reply modules/ROOT/pages/policy-mule4-detokenization.adoc]: Understood. See: df71b31

**Review by LorenaSoledad — APPROVED** [review]
LGTM 👍

**Pull request merged**